tests/Feature/ReturnRegisterDataTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+
+class ReturnRegisterDataTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->artisan('passport:install');
+
+    }
+
+    public function tokenReturning()
```

Comment:
I can't understand what is going on in this test because the name is incorrect. You should name it like: registerTest